Sources/Purchasing/StoreKitAbstractions/Test Data/TestStoreProduct.swift

```diff
@@ -99,22 +99,25 @@ public struct TestStoreProduct {
 
 }
 
+// swiftlint:disable missing_docs
+
 // Ensure consistency with the internal type
-extension TestStoreProduct: StoreProductType {
+@_spi(Internal) extension TestStoreProduct: StoreProductType {
```

Comment:
We need this because `TestStoreProduct` is public and now `StoreProductType` is `@_spi(Internal) public`